### PR TITLE
Fix GetBlocks & RC Jar Update

### PR DESCRIPTION
**Pull request by mitchej123**
Fix getBlock call on null chunks

Meh, RC jar changed; need a less brittle fix for this

### CODE REVIEW

File: src/main/java/com/mitchej123/hodgepodge/core/HodgepodgeMixinPlugin.java [hunk 1]
```diff
@@ -168,7 +168,7 @@ public void postApply(String targetClassName, ClassNode targetClass, String mixi
         ),
         WAKE_ANCHORS_ON_LOGIN("Wake up personal anchors on owner login",
             () -> config.installAnchorAlarm,
-            "Railcraft_1.7.10",
+            "Railcraft-1.7.10",
```
Comment (repo-alt) [inline src/main/java/com/mitchej123/hodgepodge/core/HodgepodgeMixinPlugin.java:171]: Huh, that's why archor wakeup broke..

Comment (mitchej123) [reply src/main/java/com/mitchej123/hodgepodge/core/HodgepodgeMixinPlugin.java:171]: Yah... I'm going to add a config option to make jars required and have it exit if they're not found.

**Review by repo-alt — APPROVED** [review]

**Comment by boubou19** [comment]
with that, we should remove lot of crashes for the officials. But it need to be tested on thermos tho

**Comment by Dream-Master** [comment]
will pack in zeta tomorrow

**Comment by Dream-Master** [comment]
merge mitch :P

**Comment by mitchej123** [comment]
> merge mitch :P

Too slow Dream!

**Pull request merged**